src/components/addNote/index.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { useRef, useEffect, FC, useState } from 'react';
+
+const AddNote: FC = () => {
+  const [showSavebutton, setshowSavebutton] = useState(false);
+  const textareaRef = useRef<HTMLTextAreaElement>(null);
+  const heightLimit = 200; // Maximum height: 200px
+
+  useEffect(() => {
+    const textarea = textareaRef.current;
+
+    if (!textarea) return;
+
+    const handleInput = () => {
+      textarea.style.height = ''; // Reset the height
+      textarea.style.height =
+        Math.min(textarea.scrollHeight, heightLimit) + 'px';
+    };
+
+    textarea.addEventListener('input', handleInput);
+
+    // Clean up the event listener
+    return () => {
+      textarea.removeEventListener('input', handleInput);
+    };
+  }, []);
+
+  return (
+    <div>
```

Comment:
enclosing div should not have 100% width. It should only have 40-50% width and should be center aligned. It should also have a box shadow. Please refer the screenshot